### DISCUSSION TITLE
2022-09-02 现在可以指定每个人推送不同的模板了，而且每个模板消息支持自定义打开一个你想要展示的网址。

### DISCUSSION THREAD

**Discussion by wangxinleo** [wangxinleo/wechat-public-account-push#93] [Announcements]
字不重要，看图。

![image](https://user-images.githubusercontent.com/28293988/188188955-92605f25-72b8-4ff5-bd7f-66aedd9b0a59.png)

**Comment by xxxie0511** [comment]
不同的人在不同的城市，天气等信息如何区分？

**Reply by wangxinleo** [reply]
有道理，我没想到，看来这个也需要区分。下次会更新

**Reply by LogicmanMrZhang** [reply]
> 有道理，我没想到，看来这个也需要区分。下次会更新

楼主能加个vx教一下吗 有偿的

**Reply by wangxinleo** [reply]
> > 有道理，我没想到，看来这个也需要区分。下次会更新
> 
> 楼主能加个vx教一下吗 有偿的

加群吧，文档有群链接。
